src/main/java/com/example/Library/controller/BooksController.java

```diff
@@ -1,11 +1,11 @@
 package com.example.Library.controller;
 
+import com.example.Library.dto.BookDTO;
 import com.example.Library.enums.BookStatus;
 import com.example.Library.exceptions.BookNotFoundByIdException;
 import com.example.Library.exceptions.IncorrectLanguageException;
-import com.example.Library.model.Book;
 import com.example.Library.model.Language;
-import com.example.Library.services.BookService;
```

Comment:
the Controller should have dependency only on the BookService interface, not the implementation